### PR TITLE
Add waterbend to the story

### PR DESCRIPTION
**Pull request by neverush**
Added waterbending to the story

### CODE REVIEW

**Comment by morosebose** [comment]
@casey-collab

File: english/waterbend/waterbend.md [hunk 1]
```diff
@@ -0,0 +1 @@
+After doing outdoor yoga for the first time, you noticed how the stream of water reflect the way you move.
```
Comment (casey-collab) [inline english/waterbend/waterbend.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by neverush** [comment]
Hi Casey, I've added a line to marshmallow.md, have a look!

**Pull request closed without merging**